pom.xml

```diff
@@ -33,9 +33,9 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>1.29.4</iot-device-client-version>
-        <iot-service-client-version>1.28.0</iot-service-client-version>
-        <iot-deps-version>0.11.2</iot-deps-version>
+        <iot-device-client-version>1.29.5</iot-device-client-version>
+        <iot-service-client-version>1.29.0</iot-service-client-version>
+        <iot-deps-version>0.11.3</iot-deps-version>
```

Comment:
wondering if the deps changes qualify for minor update or just patch?

Reply:
Just a patch since no new features were added. Some non-user facing classes were added, which historically only qualifies for patch

Reply:
Not sure if we decide to take my latest change it will be a feature add to the client SDK and this will need to be bumped

Reply:
Yep, I'll make the device client a minor bump now that you have added a new feature. Deps should stay as a patch though since you didn't add new features to the deps library in particular